client/src/App.tsx

```diff
@@ -4,6 +4,9 @@ import AppShell from "./components/AppShell.tsx";
 import ErrorElement from "./pages/ErrorElement.tsx";
 import Challenges from "./pages/Challenges.tsx";
 import Challenge from "./pages/Challenge.tsx";
+import Solutions from "./pages/Solutions.tsx";
+import PostSolution from "./pages/PostSolution.tsx";
```

Comment:
What is the difference between Solutions , PostSolution and Solution ? How does it work?

Reply:
Solutions is the page that displays all the solutions in a grid. PostSolution is the form page that gets displayed when the user clicks on post solution button in the challenge display and the Solution is the page that displays the challenge details (with comments).